app/views/places/_place.html.erb

```diff
@@ -16,8 +16,8 @@
   <td class="col col-currency"><%= place.currency&.upcase %></td>
   <td class="col col-address"><%= place.address %></td>
   <td class="col col-distance_from_center"><%= place.distance_from_center %></td>
-  <td class="col col-available_since"><%= place.available_since %></td>
-  <td class="col col-available_till"><%= place.available_till %></td>
+  <td class="col col-available_since"><%= place.available_since.strftime("%a, %B %d") %></td>
```

Comment:
Please, check in case of object has empty `available_since` and `available_till` we will retrieve an error? may be safe navigation should be used?